### PR TITLE
January Changelog

### PR DESCRIPTION
**Pull request by noanflaherty**
Check it out on staging here: https://vellum-staging.docs.buildwithfern.com/changelog#prompt-deployment-usage-tracking

### CODE REVIEW

File: fern/docs/content/change-log/2024-01.mdx [hunk 1]
```diff
@@ -0,0 +1,148 @@
+## Prompt Deployment Usage Tracking
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-01.mdx:1]: @m-abboud feel free to provide feedback to this section

File: fern/docs/content/change-log/2024-01.mdx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+<video
+  controls
+  width="100%"
+  height="500px"
+  src="https://storage.googleapis.com/vellum-public/help-docs/single-editor-mode-demo.mp4"
+></video>
+
+---
+
+## API to Execute Workflow w/o Streaming
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-01.mdx:36]: @siddseethepalli feel free to update this section. Ideally we update it once we can link to API docs.

File: fern/docs/content/change-log/2024-01.mdx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+<video
+  controls
+  width="100%"
+  height="500px"
+  src="https://storage.googleapis.com/vellum-public/help-docs/upload-download-of-function-definitions-demo.mp4"
+></video>
+
+---
+
+## Image Support for OpenAI Vision Models
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-01.mdx:70]: @dvargas92495 feel free to improve this section

File: fern/docs/content/change-log/2024-01.mdx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+*January 18th, 2024*
+
+Vellum now has API support for interacting with OpenAI's vision models, such as `gpt-4-vision-preview`.
+You can learn more about OpenAI Vision models [here](https://platform.openai.com/docs/guides/vision). Note that there is
+limited support for images in the Vellum UI at this time, but you can still use the API to interact with OpenAI Vision models.
+UI support coming soon!
+
+---
+
+## Folders
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-01.mdx:80]: @jacksonhardy feel free to update this section

File: fern/docs/content/change-log/2024-01.mdx [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+  controls
+  width="100%"
+  height="500px"
+  src="https://storage.googleapis.com/vellum-public/help-docs/folders-demo.mp4"
+></video>
+
+
+---
+
+## Support for OpenAI JSON Mode, User ID, and Seed Params
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-01.mdx:97]: @dvargas92495 feel free to update this section

File: fern/docs/content/change-log/2024-01.mdx [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+*January 9th, 2024*
+
+You can now clone a Workflow Scenario to create a new Scenario based on an existing one. This is useful when you want to
+create a new Scenario that is similar to an existing one, but with some changes.
+
+![Clone Workflow Scenario](https://storage.googleapis.com/vellum-public/help-docs/clone-workflow-scenario.png)
+
+---
+
+## API Key Metadata
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-01.mdx:118]: @m-abboud feel free to make changes to this section

File: fern/docs/content/change-log/2024-01.mdx [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+You can now find the following actions at the top-level of the Workflow and Prompt Deployment Execution pages:
+- **Save as Scenario:** Useful for saving an edge case seen in production as a Scenario for qualitative eval.
+- **Save as Test Case:** Useful for saving an edge case seen in production to your bank of Test Cases for quantitative eval.
+- **View Details:** Drill in to see specifics about that specific Execution.
+
+![Top-Level Workflow Execution Actions](https://storage.googleapis.com/vellum-public/help-docs/top-level-workflow-execution-actions.png)
+
+---
+
+## Improved Error Messages in Code & API Nodes
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-01.mdx:141]: @siddseethepalli feel free to make changes to this section

**Review by siddseethepalli — APPROVED** [review]

File: fern/docs/content/change-log/2024-01.mdx [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+Going forward, Vellum will now keep track of the token utilization of your Prompt Deployments. You can keep tabs on
+input, output, and total tokens used per request.
+
+![Token Count Row Data](https://storage.googleapis.com/vellum-public/help-docs/token-count-row-data.png)
+
+You can also view this data in aggregate in the Monitoring tab.
+
+![Token Count Time-Series Data](https://storage.googleapis.com/vellum-public/help-docs/token-count-time-series-data.png)
+
+This is a precursor to more advanced usage and billing features coming down the road.If there's more you'd like to see
```
Comment (m-abboud) [inline fern/docs/content/change-log/2024-01.mdx:13]: Missing space after period
```suggestion
This is a precursor to more advanced usage and billing features coming down the road. If there's more you'd like to see
```

**Review by m-abboud — APPROVED** [review]
Looks good to me :)

File: fern/docs/content/change-log/2024-01.mdx [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+## Image Support for OpenAI Vision Models
+*January 18th, 2024*
+
+Vellum now has API support for interacting with OpenAI's vision models, such as `gpt-4-vision-preview`.
+You can learn more about OpenAI Vision models [here](https://platform.openai.com/docs/guides/vision). Note that there is
+limited support for images in the Vellum UI at this time, but you can still use the API to interact with OpenAI Vision models.
+UI support coming soon!
+
+---
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-01.mdx:78]:
````suggestion
Here's a quick example on how to send an image to the model, using our python sdk:

```python
image_link = "https://storage.googleapis.com/vellum-public/help-docs/add_prompt_block_button.png"
response = client.execute_prompt(
    prompt_deployment_name="github-loom-demo",
    inputs=[
        PromptDeploymentInputRequest_ChatHistory(
            name="$chat_history",
            value=[
                ChatMessageRequest(
                    role=ChatMessageRole.USER,
                    content={
                        "type": "ARRAY",
                        "value": [
                            {"type": "STRING", "value": "What's in this image?"},
                            {"type": "IMAGE", "value": {"src": image_link}},
                        ],
                    },
                )
            ],
            type=VellumVariableType.CHAT_HISTORY,
        ),
    ],
)
print(response.outputs[0].value)
```

---
````

File: fern/docs/content/change-log/2024-01.mdx [hunk 10]
```diff
@@ -0,0 +1,148 @@
...
+
+<video
+  controls
+  width="100%"
+  height="500px"
+  src="https://storage.googleapis.com/vellum-public/help-docs/folders-demo.mp4"
+></video>
+
+
+---
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-01.mdx:95]:
```suggestion
---

## Support for Google Gemini Safety Settings
*January 12th, 2024*

There is now native support for setting the `safetySettint` parameters in Google Gemini prompts. You can learn more about how these parameters are used by Google in their docs [here](https://ai.google.dev/api/rest/v1beta/SafetySetting).

![Gemini Custom Parameters](https://storage.googleapis.com/vellum-public/help-docs/gemini-custom-params.png)

---
```

**Review by dvargas92495 — APPROVED** [review]
